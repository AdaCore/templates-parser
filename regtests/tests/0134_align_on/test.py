from test_support import *

gprbuild('alo')
run('alo')
