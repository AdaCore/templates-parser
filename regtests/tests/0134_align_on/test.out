Some Text : To Check Aligment

Test 1

   Part1    :    process Part1.final;
   Part2    :    process Part2.final;

Test 2

   Part1 : process Part1.final;
   Part2 : process Part2.final;

Test 3

   Part2 : process Part2.final;
   Part1 : process Part1.final;
